plugin-maven/build.gradle

```diff
@@ -55,17 +28,20 @@ def mvnw(String args) {
 	}
 }
 
-String libVersion = version.endsWith('-SNAPSHOT') ?
-		rootProject.spotlessChangelog.versionNext :
-		rootProject.spotlessChangelog.versionLast
+apply plugin: 'de.benediktritter.maven-plugin-development'
+mavenPlugin {
+	name = 'Spotless Maven Plugin'
+	artifactId = project.artifactIdMaven
+	description = project.description
+}
+
+String VER_MAVEN_API = '3.0'
+String VER_ECLIPSE_AETHER = '1.1.0'
+String VER_MUSTACHE = '0.9.10'
```

Comment:
Perhaps inline this constant? It is only used for a single `testImplementation` dependency

Reply:
Good call! Done.